src/components/elements/off-canvas-menu.js

```diff
@@ -239,6 +240,20 @@ export const OffCanvasMenuWrapper = (props) => {
                                 to="/dtrader/"
                             />
                         </Flex>
+                        {!is_eu_country && (
+                            <Flex mb="2rem">
+                                <NavCard
+                                    aria_label="DerivGo"
+                                    icon={() => <img src={DerivGo} alt="" width="32" height="32" />}
```

Comment:
Don't we need to provide an alt attr here?

Reply:
I made this NavCard as others presented here, they also doesn't have alt values.

Reply:
Yes it seems there is an SEO reason about this

https://github.com/binary-com/deriv-com/pull/1956